File: TestProjectWebApi/Models/Address.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace TestProjectWebApi.Models
{
    public class Address
    {
        [Required] public string Address1 { get; set; }
        [Required] public string Address2 { get; set; }
        [Required] public string City { get; set; }
        [Required] public string State { get; set; }
        [Required] public int ZipCode { get; set; }
        [Required] public string Country { get; set; } = "India";
        public override string ToString() => $"{Address1},{Address2},{City},{State},{Country}-{ZipCode}";
    }
}

File: TestProjectAngluar/TestProjectAngluar/src/app/models/Address.ts
export class Address
{
    address1:string;
    address2:string;
    city:string;
    state:string;
    zipCode:number;
    constructor(address1?,address2?,city?,state?,zip?)
    {
        this.address1=address1;
        this,address2=address2;
        this.city=city;
        this.state=state;
        this.zipCode=zip;
    }
}